modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/SW360Client.java

```diff
@@ -65,6 +66,11 @@ public SW360Client(boolean proxyUse, String proxyHost, int proxyPort) {
     }
 
     protected <T> ResponseEntity<T> doRestPATCH(String url, HttpEntity<?> httpEntity, ParameterizedTypeReference<T> responseType) {
-        return doRestCall(url, HttpMethod.PATCH, httpEntity, responseType);
+        RestTemplate restTemplate = new RestTemplate();
+
+        HttpComponentsClientHttpRequestFactory requestFactory = new HttpComponentsClientHttpRequestFactory();
```

Comment:
Why not instead change the `SimpleClientHttpRequestFactory requestFactory` the constructor to `HttpComponentsClientHttpRequestFactory requestFactory`?

Reply:
`HttpComponentsClientHttpRequestFactory` doesn't have a possibility to set a proxy

Reply:
This means that this fix only works without a proxy?

Reply:
Maybe throw an unsupported operation exception, if patch is requested with a proxy